hudi-common/src/main/java/org/apache/hudi/common/bloom/filter/InternalDynamicBloomFilter.java

```diff
@@ -217,12 +214,8 @@ public void readFields(DataInput in) throws IOException {
    * Adds a new row to <i>this</i> dynamic Bloom filter.
    */
   private void addRow() {
-    org.apache.hadoop.util.bloom.BloomFilter[] tmp = new org.apache.hadoop.util.bloom.BloomFilter[matrix.length + 1];
-
-    for (int i = 0; i < matrix.length; i++) {
-      tmp[i] = matrix[i];
-    }
-
+    BloomFilter[] tmp = new BloomFilter[matrix.length + 1];
+    System.arraycopy(matrix, 0, tmp, 0, matrix.length);
     tmp[tmp.length - 1] = new org.apache.hadoop.util.bloom.BloomFilter(vectorSize, nbHash, hashType);
```

Comment:
So org.apache.hadoop.util.bloom.BloomFilter could be also changed to BloomFilter?